motioneye/meyectl.py

```diff
@@ -24,33 +24,40 @@
 import pipes
 import sys
 
-# ŝarĝante tradukojn
-locale.setlocale(locale.LC_ALL, '')
-lingvo = 'eo'
-traduction = None
-pathname = os.path.dirname(__file__)
-try:
-    gettext.find('motioneye', pathname + '/locale')
-    traduction = gettext.translation('motioneye', pathname + '/locale')
-    traduction.install()
-except:
-    traduction = gettext
-    gettext.install('motioneye')
-
-file = gettext.find('motioneye', pathname + '/locale')
-if file:
-    lgrpath = len(pathname)
-    lingvo = file[lgrpath + 8 : lgrpath + 10]
-else:
-    lingvo = 'eo'
-# logging.info(_('lingvo : ') + lingvo)
+from motioneye import config, settings
+
+_LOG_FILE = 'motioneye.log'
 
 # make sure motioneye is on python path
 sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
-from motioneye import settings
+lingvo = 'eo'
+traduction = None
 
-_LOG_FILE = 'motioneye.log'
+# ŝarĝante tradukojn
+def load_l10n():
+    global lingvo, traduction
```

Comment:
As of the CodeFactor/Pylint annotation: There is no way around global variables here, right? At least I can imagine it when those need to be enabled across all script for strings to be translated.